script/Arcueid/Day 7/07_00_ARC07_3F.txt

```diff
@@ -161,7 +161,7 @@ Seemingly amused,
 [sha:71112ebf223996b999ac5e165254952fd1f5f999]{
 -- Page 348, Offset 17437.
 -- 下級の死者だったけど、それでも放っておくと人間を殺して力を蓄えられてしまうから。
-He was just a low rank Dead,
+He was just a low-rank Dead,
```

Comment:
He was just a low-ranked Dead,

Reply:
Maybe its just a spoken dropping of the "-ed", but I can't imagine saying this as "low-ranked"